cpp/ql/test/query-tests/Likely Bugs/Memory Management/AllocaInLoop/AllocaInLoop1.cpp

```diff
@@ -0,0 +1,90 @@
+// semmle-extractor-options: --clang
+struct vtype {
+	int i1, i2;
+};
+extern int w1, w2;
+
+#ifdef _MSC_VER
+#define restrict __restrict
+#else
+#define restrict __restrict__
+#endif
+
+void *__builtin_alloca(int sz);
```

Comment:
Please change `int` to `size_t`.

Reply:
I respectfully disagree.  __builtin_alloca() is a compiler intrinsic, and as such cannot depend on extrinsic types.

Reply:
It's listed as taking a `size_t` argument on https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html, but I see your point. How about changing `int` to `unsigned long`, then?